src/test/integration/StrategyVaultV2.int.sol

```diff
@@ -24,6 +24,12 @@ abstract contract StrategyVaultV2_IntegrationTest is CommonVaultTest {
         vm.prank(governance);
         vault.upgradeTo(address(impl));
         asset = ERC20(vault.asset());
+        // vm.prank(governance);
+        // StrategyVaultV2(address(vault)).setTvlCap(type(uint256).max);
+    }
+
+    function _getRemainingVaultTvlCap() internal virtual override returns (uint256) {
```

Comment:
Could we update `StrategyVault.t.sol` to use a `StrategyVaultV2`? That test file has a tvl cap test.